doc/py_interconnectivity.rst

```diff
@@ -0,0 +1,68 @@
+.. _pyinterconnectivity:
+
+Interconnectivity
+#################
+
+.. class:: connection
+
+    Describes a connection between two cells:
+    Defined by source and destination end points (that is pre-synaptic and post-synaptic respectively),
```

Comment:
Join sentence fragment above with this line, i.e. 'Describes a connection between two cells, defined by ...'